src/components/Board.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import './App.css';
+import DragArea from '../dnd/DragArea';
+
+const CREATE_TASKS = (count, containerID) => {
+  const tasks = {};
+  for (let i = 0; i <= count; i++) {
+    const id = Math.random().toString(36).substring(7);
+    tasks[id] = { ID: id, containerID };
+  }
+
+  return tasks;
+};
+
+const createContent = () => {
```

Comment:
Move out and all mock data below

Just import what's needed

Reply:
moved to mocks/BoardMocks.js